types/express-ws/index.d.ts

```diff
@@ -31,10 +31,10 @@ declare namespace expressWs {
     interface Instance {
         app: Application;
         applyTo(target: RouterLike): void;
-        getWss(): ws.Server;
+        getWss(): ws.WebSocketServer;
```

Comment:
`ws` defines `WebSocketServer` as a synonym of `Server` by declaring a `const` and an `interface`. What's the advantage of switching? Based only on the types, I'd prefer `Server` since it's the original. But I know nothing about `ws` except what I got from reading index.d.ts.